File: squid_proxy_service/examples/java-v2/src/main/java/com/chariotsolutions/example/NoProxy.java
package com.chariotsolutions.example;

import software.amazon.awssdk.services.sts.StsClient;
import software.amazon.awssdk.services.sts.model.*;

/**
 *  Retrieves the caller's identity using a default client.
 */
public class NoProxy
{
    public static void main(String[] argv)
    throws Exception
    {
        StsClient client = StsClient.create();
        
        GetCallerIdentityRequest request = GetCallerIdentityRequest.builder().build();
        GetCallerIdentityResponse response = client.getCallerIdentity(request);
        
        System.out.println("account: " + response.account());
        System.out.println("user id: " + response.userId());
        System.out.println("arn:     " + response.arn());
    }
}

File: squid_proxy_service/examples/javascript/no_proxy.js
var AWS = require('aws-sdk');

var STS = new AWS.STS();
STS.getCallerIdentity({}, function(err, data) {
    if (err) {
        console.log("error: " + err)
    }
    else {
        console.log("User:    " + data.UserId)
        console.log("Account: " + data.Account)
        console.log("Arn:     " + data.Arn)
    }
});
